src/js/common/schemaform/definitions/dateRange.js

```diff
@@ -17,7 +17,7 @@ export function uiSchema(from = 'From', to = 'To', rangeError = 'To date must be
       validateDateRange
     ],
     'ui:errorMessages': {
-      dateRange: rangeError,
+      pattern: rangeError,
```

Comment:
I'm curious about the reason for this change. This is the error key used by JSON Schema when a field fails a regex pattern match. It won't cause any problems, but it seems less clear to use it here?